ci/tasks/check-sns-topic-unsubscribers.yml

```diff
@@ -3,6 +3,12 @@ image_resource:
    type: registry-image
    source:
     repository: governmentdigitalservice/pay-concourse-runner
+params:
+  AWS_ACCESS_KEY_ID:
+  AWS_SECRET_ACCESS_KEY:
+  AWS_SESSION_TOKEN:
+  AWS_REGION:
+  AWS_DEFAULT_REGION:
```

Comment:
I don't think we need to include these here if they're not referred to in the script? We don't for some of the other tasks where Concourse assumes a role earlier in the pipeline (e.g. [scale-fargate-service](https://github.com/alphagov/pay-ci/blob/master/ci/tasks/scale-fargate-service.yml) which is used in the [perf-tests pipeline](https://github.com/alphagov/pay-ci/blob/master/ci/pipelines/perf-tests.yml)). If it doesn't work without them though, there's no harm in leaving them in 😸

Reply:
It works fine without them!
https://pay-cd.deploy.payments.service.gov.uk/teams/pay-dev/pipelines/infra-drift-detector/jobs/check-for-zendesk-unsubscribers/builds/8